index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello-Microverse</title>
```

Comment:
Please ensure that your .gitignore file is listing `test.md` file and `node_modules` directory (as it can be added by accident while setting up Linters).